tests/tck/features/go/GroupbyLimit.IntVid.feature

```diff
@@ -263,10 +263,10 @@ Feature: Groupby & limit Sentence
       YIELD $$.player.name AS name, $$.player.age AS dst_age, $$.player.age AS src_age, like.likeness AS likeness
       | GROUP BY $-.name YIELD $-.name AS name, SUM($-.dst_age) AS sum_dst_age, AVG($-.dst_age) AS avg_dst_age,
       MAX($-.src_age) AS max_src_age, MIN($-.src_age) AS min_src_age, BIT_AND(1) AS bit_and, BIT_OR(2) AS bit_or, BIT_XOR(3) AS bit_xor,
-      COUNT($-.likeness), COUNT_DISTINCT($-.likeness)
```

Comment:
I think you should keep it for compatibility.

Reply:
`COUNT_DISTINCT` is same as `COUNT(DISTINCT)`, and it's just consistent with cypher.

Reply:
Yes I means you should keep compatibility with previous version in syntax.

Reply:
You could add COUNT(DISTINCT) but keep COUNT_DISTINCT in later several version.

Reply:
ACK. But, `COUNT_DISTINCT` is not supported a long time ago.